external-providers/generic-external-provider/go.mod

```diff
@@ -38,5 +38,3 @@ require (
 	google.golang.org/protobuf v1.30.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
-
-replace github.com/konveyor/analyzer-lsp => ../../
```

Comment:
I think we want to leave this, right?

Reply:
I think that was automatically removed, when i ran `go mod tidy`. I will add it back.